crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -281,6 +281,41 @@ def prepare(options = {})
 
       protected
 
+      def upgrade_first_cluster_node(node_name, next_node_name)
+        save_upgrade_state("Starting the upgrade of node #{node_name}")
+        node_api = Api::Node.new node_name
+        next_node_api = Api::Node.new next_node_name
+        # upgrade the first node
+        return false unless node_api.upgrade
+
+        # Explicitly mark the first node as cluster founder
+        # and in case of DRBD setup, adapt DRBD config accordingly.
+        unless Api::Pacemaker.set_node_as_founder node_name
+          save_error_state("Changing the cluster founder to #{node_name} has failed")
+          return false
+        end
+        # remove pre-upgrade attribute, so the services can start
+        return false unless next_node_api.disable_pre_upgrade_attribute_for node_name
```

Comment:
Again `disable_pre_upgrade_attribute_for` does it. (But we'll need to revisit those save_error_state parts later, currently it only logs)